README.adoc

```diff
@@ -216,7 +216,7 @@ $ docker run -p 9000:9000 myorg/myapp --server.port=9000
 2019-10-29 09:30:19.751  INFO 1 --- [           main] o.s.b.web.embedded.netty.NettyWebServer  : Netty started on port(s): 9000
 ```
 
-Note the use of `${0}` for the "command" (in this case the first program argument) and `${@}` for the "command arguments" (the rest of the program arguments). If you use a script for the entry point, then you don't need the `${0}` (that would be `/app/run.sh` in the example above). Example:
+Note the use of `${0}` for the "`command`" (in this case the first program argument) and `${@}` for the "`command arguments`" (the rest of the program arguments). If you use a script for the entry point, then you don't need the `${0}` (that would be `/app/run.sh` in the example above). Example:
```

Comment:
We use backticks and quotation marks to tell Asciidoctor to create curly quotation marks. We try to use regular quotation marks only within code.